.github/workflows/publish-docker-image.yml

```diff
@@ -26,15 +36,34 @@ jobs:
           password: ${{ secrets.dockerhub-password }}
 
       - name: Extract metadata (syslog-ng version) for Docker
-        id: meta
+        if: inputs.pkg-type == 'stable'
+        id: stable-tags
         uses: docker/metadata-action@v3
         with:
-          images: balabit/syslog-ng
+          images: ${{ env.DOCKER_IMAGE_NAME }}
           tags: type=match,pattern=syslog-ng-(.*),group=1
 
+      - name: Compose Docker image tags
+        id: tags
+        run: |
+          if [[ '${{ inputs.pkg-type }}' = 'stable' ]]; then
+            TAGS='${{ steps.stable-tags.outputs.tag }}'
+          elif [[ '${{ inputs.pkg-type }}' = 'nightly' ]]; then
+            VERSION="$(scripts/version.sh)"
+            DATE="$(date +%Y%m%d)"
+            TAGS="${DOCKER_IMAGE_NAME}:nightly-${VERSION}-${DATE},${DOCKER_IMAGE_NAME}:nightly-latest"
```

Comment:
Can we use just `nightly` instead of `nightly-latest`?
I think the phrase `nightly` indicates clearly that it is created each night, `latest` may just cause confusion.

date-tagging nightly releases can be useful, but in that case, we should probably add a cleanup mechanism as well.
For example, can you keep the latest 14 nightly releases and remove the older ones?
(Or alternatively, we can start with just the undated `nightly` tag, and add a N-day history later)

Reply:
I quickly checked the settings on DockerHub, and we do not have the right plan to auto remove old tags, so sure, let's just keep it `nightly` for now, it still brings a lot of benefits.

Reply:
Fixed.

Reply:
Awesome, thanks :)